ES/ESmy001.xml

```diff
@@ -363,129 +751,143 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                            <note/>
                            <ab type="ruling"/>
                            <ab type="pricking">Pricking and ruling are visible.</ab>
-                           <ab type="ruling" subtype="pattern"> Ruling pattern:
-                              1A-1A-1A1A/0-0/0-0/C.</ab>
-                           <ab type="pricking"> Primary pricks are partly visible (s. <locus target="#43r">43r</locus>).</ab>
-                           <ab type="pricking"> Ruling pricks are visible.</ab>
-                           <ab type="ruling"> The upper line is written below the ruling.</ab>
-                           <ab type="ruling"> The bottom line is written above the ruling.</ab>
+                           <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C.</ab>
+                           <ab type="pricking">Primary pricks are partly visible (see <locus target="#43r">43r</locus>).</ab>
+                           <ab type="pricking">Ruling pricks are visible.</ab>
+                           <ab type="ruling">The upper line is written below the ruling.</ab>
+                           <ab type="ruling">The bottom line is written above the ruling.</ab>
                         </layout>
                         <layout>
                            <ab type="punctuation" subtype="Executed">not quite regularly;</ab>
                            <ab type="punctuation" subtype="Usage">not always logical.</ab>
                            <ab type="punctuation">
                               <list>
-                                 <item xml:id="div1"> Dividers:</item>
-                                 <item xml:id="div2"> I. Colon. </item>
-                                 <item xml:id="div3"> II. Four dot asterisk (e.g. fols.: 18va,
+                                 <item xml:id="div1">Dividers:</item>
```

Comment:
remove item line 765 (`<item xml:id="div1">Dividers:</item>`) - the subtype dividers should be added to ab above

Reply:
like this?

Reply:
The punctuation signs will not be in the catalogue.

Reply:
no, it should be 
<ab type="punctuation" subtype="Dividers">
                              <list>
                                 
                                 <item xml:id="div1"> I. Colon</item>
<item xml:id="div2"> II. Four dot asterisk ...........................</item>
</ab>

Reply:
Hope now correct.